src/app/(app)/payment.tsx

```diff
@@ -3,32 +3,40 @@ import { useRef } from 'react';
 
 import type { BuyFormSubmitHandler } from '@/components/payment/buy-form';
 import { BuyForm } from '@/components/payment/buy-form';
-import { Button, SafeAreaView } from '@/ui';
+import { Button, SafeAreaView, ScrollView, View } from '@/ui';
 
 export default function Payment() {
   const buyFormRef = useRef<BuyForm>(null);
 
   const onSubmitHandler: BuyFormSubmitHandler = () => {};
 
   return (
-    <SafeAreaView className="flex-1">
-      <BuyForm ref={buyFormRef} onSubmitHandler={onSubmitHandler} />
-      <Button
-        label="Buy"
-        className="h-12"
-        textClassName="font-bold text-base"
-        onPress={() => {
-          buyFormRef.current?.submit();
-        }}
-      />
-      <Button
-        variant="outline"
-        label="Cancel"
-        className="mb-8"
-        onPress={() => {
-          router.back();
-        }}
-      />
+    <SafeAreaView className="flex-1 bg-light_background">
+      <ScrollView className="-mb-12">
+        <BuyForm
+          ref={buyFormRef}
+          onSubmitHandler={onSubmitHandler}
+          disabled={false}
```

Comment:
Generally disabled is a prop that is by default false, the fact that this value needs to be passed it's a little weird, check if its necessary otherwise it would be better to configure disabled default value ot be false